examples/player/simid_player.html

```diff
@@ -132,22 +138,38 @@
       <tr>
         <td>Ad type</td>
         <td>
-          <input type="radio" id="linear_ad" name="ad_type" checked> <label for="linear_ad" class="ad_type_label">Linear
+          <input type="radio" id="linear_ad" name="ad_type" onclick="toggleInputs()" checked> <label for="linear_ad" class="ad_type_label">Linear
         Ad</label>
-        <input type="radio" id="nonlinear_ad" name="ad_type"> <label for="nonlinear_ad" class="ad_type_label">Non-Linear
+        <input type="radio" id="nonlinear_ad" name="ad_type" onclick="toggleInputs()"> <label for="nonlinear_ad" class="ad_type_label">Non-Linear
           Ad</label>
         </td>
       </tr>
-      <tr>
-        <td>Interactive Creative File</td>
-        <td><input type="text" id="creative_url" size = 30 
-           value="../creatives/simid_overlay.html">
+      <tr id="creative_dimensions" class="hidden">
+        <td>Creative Dimensions</td>
+        <td>
+          <label for="x_val">X: </label>
+          <input type="number" id="x_val" value="5" min="0" class="resize_inputs">
+
+          <label for="y_val">Y: </label>
+          <input type="number" id="y_val" value="5" min="0" class="resize_inputs">
+
+          <label for="width">Width: </label>
+          <input type="number" id="width" value="500" min="0" class="resize_inputs">
```

Comment:
Just curious -- do these default values signify anything or are they just random?

Reply:
we put them there so we didn't have to constantly input them manually

Reply:
these numbers are not too small/not too big and just hold the buttons and the input fields